### PR TITLE
feat(payment): PAYPAL-3377 Add support of braintree.ach in payments API

### PR DESCRIPTION
**Pull request by alstarodubtcev**
# [PAYPAL-3377]

## What changed?

* Added bank account instrument to payments API doc.
* Added Braintree ACH block to payments overview. 

## Release notes draft

* 

## Anything else?

ping {names}

[PAYPAL-3377]: https://bigcommercecloud.atlassian.net/browse/PAYPAL-3377?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/payments/payments-api-overview.mdx [hunk 1]
```diff
@@ -116,6 +116,12 @@ The store must:
 
 Use the active MSF-enabled store's control panel to enable charging stored credit cards. Navigate to **Settings ›  Setup > Payments** and click the tab for your payment gateway. Toggle the switch to enable Stored Credit Cards and click **Save**. For more on enabling stored payment methods, see [Enabling Stored Payment Methods](https://support.bigcommerce.com/s/article/Enabling-Stored-Credit-Cards).
 
+### Using Braintree ACH stored bank accounts
+
+Additionally to previous prerequisites, in order to use Braintree ACH in payments API your store need to:
```
Comment (bc-traciporter) [inline docs/api-docs/payments/payments-api-overview.mdx:121]:
```suggestion
In addition to the [prerequisites](/docs/store-operations/payments#prerequisites-for-charging-stored-payment-instruments) above, to use Braintree ACH in payments API, your store needs to:
```

File: docs/api-docs/payments/payments-api-overview.mdx [hunk 2]
```diff
@@ -116,6 +116,12 @@ The store must:
...
+* have stored ACH accounts enabled.
+* contact support to enable this method in API.
```
Comment (bc-traciporter) [inline docs/api-docs/payments/payments-api-overview.mdx:123]:
```suggestion
* Have this method enabled in the API. (contact support)
```

File: docs/api-docs/payments/payments-api-overview.mdx [hunk 3]
```diff
@@ -116,6 +116,12 @@ The store must:
 
 Use the active MSF-enabled store's control panel to enable charging stored credit cards. Navigate to **Settings ›  Setup > Payments** and click the tab for your payment gateway. Toggle the switch to enable Stored Credit Cards and click **Save**. For more on enabling stored payment methods, see [Enabling Stored Payment Methods](https://support.bigcommerce.com/s/article/Enabling-Stored-Credit-Cards).
 
+### Using Braintree ACH stored bank accounts
+
+Additionally to previous prerequisites, in order to use Braintree ACH in payments API your store need to:
+* have stored ACH accounts enabled.
```
Comment (bc-traciporter) [inline docs/api-docs/payments/payments-api-overview.mdx:122]:
```suggestion
* Have stored ACH accounts enabled.
```

**Review by bc-traciporter — COMMENTED** [review]
I left a few comments.

**Review by bc-tgomez — APPROVED** [review]
Looks good!

**Pull request merged**